yb-voyager/src/query/queryparser/ddl_processor.go

```diff
@@ -399,6 +399,15 @@ func (t *Table) PrimaryKeyColumns() []string {
 	return []string{}
 }
 
+func (t *Table) GetColumnType(colName string) string {
+	for _, col := range t.Columns {
```

Comment:
maybe add a todo here, this seems suboptimal to loop through. We should have a map or something stored in the table to help with such operations.